spec/lib/guardian/topic_guardian_spec.rb

```diff
@@ -95,12 +95,12 @@
 
     it "returns true when tl4 can delete posts and topics" do
       expect(Guardian.new(tl4_user).can_see_deleted_topics?(topic.category)).to eq(false)
-      SiteSetting.tl4_delete_posts_and_topics = true
+      SiteSetting.delete_all_posts_and_topics_allowed_groups = 14
```

Comment:
I think using `Group::AUTO_GROUPS[:trust_level_4]` should be preferred, otherwise the 14 is going to be confusing to future passerby.

Reply:
Great idea! 👍